CHANGELOG.md

```diff
@@ -1,6 +1,23 @@
 # Changelog
 
 ## [Unreleased]
+## [0.2.169](https://github.com/rust-lang/libc/compare/0.2.168...0.2.169) - 2024-12-18
+
+### Added
+
+- Update c_char type ([#4195](https://github.com/rust-lang/libc/pull/4195))
```

Comment:
I think this should be included in the “Fixed” group to which “Fix c_char for various targets” belongs.

Reply:
Indeed, unfortunately the automatic changelogs don't do too well for anything other than "conventional commits". I always clean this up before merging the release.